### PR TITLE
12402 income limits mocked endpoint module setup

### PR DESCRIPTION
**Pull request by jtmst**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- Creates a new module for income limits
- returns mocked response
- conforms to existing endpoint standards (validation/error handling)
- Does not return actual data until we have established oracle DB connection

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12402

## Testing done

- Local testing
## What areas of the site does it impact?
- new module, added to gem file

sample URL for testing: http://localhost:3000/income_limits/v1/limitsByZipCode/12345/2022/2

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: modules/income_limits/spec/requests/v1/limits_by_zip_code_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Income limits endpoints', type: :request do
+  describe '#get limitsByZipCode/:zip/:year/:dependents' do
+    it 'returns sample data' do
+      get '/income_limits/v1/limitsByZipCode/12345/2022/2'
+      expect(response).to have_http_status(:ok)
+      JSON.parse(response.body)
```
Comment (maxx1128) [inline modules/income_limits/spec/requests/v1/limits_by_zip_code_spec.rb:10]: **Question:** Is this checking that the response body exists as part of the tests? It's unclear, and it should try to check the data structure as well, such as the JSON having all the expected keys.

**Review by maxx1128 — APPROVED** [review]
I have one question that could have some extra changes for this PR. But everything else looks good and I was able to confirm the endpoint works for me locally. So even with the potential change in the test, it is still good to merge.

**Review by jeremy6d — APPROVED** [review]
LGTM

File: modules/income_limits/spec/requests/v1/limits_by_zip_code_spec.rb (hunk 1 above)
Comment (jtmst) [reply modules/income_limits/spec/requests/v1/limits_by_zip_code_spec.rb:10]: I was holding off on adding those tests until we have connection to the oracle DB and can see the actual return data shape

**Pull request merged**